lblod/ember-rdfa-editor-lblod-plugins
# IRGN - add traffic sign prefix

### Overview
Governments want to be able to see the type of traffic sign prefixed in the generated rdfa when adding a traffic measure.

##### connected issues and PRs:
https://binnenland.atlassian.net/browse/GN-4882

### How to test/reproduce
Try to add a traffic sign. Depending on the type of sign the prefix should change.

### Challenges/uncertainties
Not super familiar with IRGN and the editor, so all feedback is welcome. Based on the ticket description, this is what i understood that needed to happen.


### Checks PR readiness
